charts/nri-kube-events/templates/configmap.yaml

```diff
@@ -18,3 +18,16 @@ metadata:
   namespace: {{ .Release.Namespace }}
   labels:
 {{ include "nri-kube-events.labels" . | indent 4 }}
+{{ if .Values.config }}
+---
+apiVersion: v1
+kind: ConfigMap
+data:
+    newrelic-infra.yml: |
+{{ toYaml .Values.config | indent 6 }}
+metadata:
+  name: {{ template "nri-kube-events.fullname" . }}-agent-config
+  namespace: {{ .Release.Namespace }}
+  labels: 
+{{ include "nri-kube-events.labels" . | indent 4 }}
+{{ end }}
```

Comment:
If you forget about the `\n` at the end of the file yo go to POSIX hell, where people build tools that do many things and do them wrong :)

Reply:
I am a bit confused by the comment, I guess you want a `\n`, right? :)